addons/parachute/functions/fnc_handleFailureChance.sqf

```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+/*
+ * Author: JoramD
+ * Handles percentage chance parachute failure.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, vehicle player] call FUNC(handleFailureChance);
+ *
+ * Public: No
+ */
+
+params ["_unit", "_vehicle"];
+
+if !(_vehicle isKindOf "ParachuteBase" || {GVAR(failureChance) > 0}) exitWith {};
```

Comment:
I personally would flip those conditions because checking if a value is bigger than 0 should be faster than checking if a object is a type
```suggestion
if !(GVAR(failureChance) > 0 || {_vehicle isKindOf "ParachuteBase"}) exitWith {};
```

Reply:
Thank you for the suggestion!